tests/bluetooth/bsim_bt/bsim_test_eatt/src/main_autoconnect.c

```diff
@@ -0,0 +1,52 @@
+/* main_connect.c - Application main entry point */
+
+/*
+ * Copyright (c) 2022 Nordic Semiconductor
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "common.h"
+
+static void test_peripheral_main(void)
+{
+	peripheral_setup_and_connect();
+
+	while (num_eatt_channels < CONFIG_BT_EATT_MAX) {
+		k_sleep(K_MSEC(10));
+	}
+
+	peripheral_disconnect();
+
+	PASS("EATT Peripheral tests Passed\n");
+}
+
+static void test_central_main(void)
+{
+	central_setup_and_connect();
+
+	while (num_eatt_channels < CONFIG_BT_EATT_MAX) {
+		k_sleep(K_MSEC(10));
+	}
+
+	central_disconnect();
+
+	PASS("EATT Central tests Passed\n");
+}
+
+static const struct bst_test_instance test_def[] = { { .test_id = "peripheral_autoconnect",
+						       .test_descr = "Peripheral autoconnect",
+						       .test_post_init_f = test_init,
+						       .test_tick_f = test_tick,
+						       .test_main_f = test_peripheral_main },
+						     { .test_id = "central_autoconnect",
+						       .test_descr = "Central autoconnect",
+						       .test_post_init_f = test_init,
+						       .test_tick_f = test_tick,
+						       .test_main_f = test_central_main },
+						     BSTEST_END_MARKER };
```

Comment:
I'm seeing this way of indenting structs a lot; is this some auto-clang format stuff? 

```suggestion
static const struct bst_test_instance test_def[] = { 
	{ 
		.test_id = "peripheral_autoconnect",
		.test_descr = "Peripheral autoconnect",
		.test_post_init_f = test_init,
		.test_tick_f = test_tick,
		.test_main_f = test_peripheral_main 
	},
	{ 
		.test_id = "central_autoconnect",
		.test_descr = "Central autoconnect",
		.test_post_init_f = test_init,
		.test_tick_f = test_tick,
		.test_main_f = test_central_main 
	},
	BSTEST_END_MARKER 
};
```

Reply:
Yes, this is the autoformat that chooses a weird format.

Reply:
Adding a trailing comma makes clang-format behave.